indigo/indigo/src/main/scala/indigo/shared/platform/SceneProcessor.scala

```diff
@@ -143,9 +149,12 @@ final class SceneProcessor(
 
     val sceneBlend = scene.blendMaterial.getOrElse(BlendMaterial.Normal).toShaderData
 
+    displayObjectConverter.purgeEachFrame()
```

Comment:
Looking at the `QuickCache` code, I'm now wondering, isn't it faster to make `cache` inside that class a `var`, then just simply drop the reference to that object, and create an empty one? Depending on the number of cached elements, it seems that purging it iterates over the entire set of keys removing them...

Reply:
Good question. This cache here is a "per frame" cache, so the number of entries is likely to be low. Thus we're trading off iterating and removing vs reallocation+GC. I couldn't say which is better, maybe worth trying to benchmark it though (benchmarks are set up, just needs a new test).